source/includes/steps-fle-convert-to-a-remote-master-key-gcp.yaml

```diff
@@ -306,6 +332,43 @@ content: |
               var dataKeyIdBase64 = Convert.ToBase64String(GuidConverter.ToBytes(dataKeyId, GuidRepresentation.Standard));
               Console.WriteLine($"DataKeyId [base64]: {dataKeyIdBase64}");
 
+        .. tab::
+           :tabid: go
+
+           .. code-block:: go
+
+              clientEncryptionOpts := options.ClientEncryption().SetKeyVaultNamespace(keyVaultNamespace).SetKmsProviders(kmsProviders)
+              keyVaultClient, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(uri))
+              if err != nil {
+                  log.Panicf("Client encryption connect error %v", err)
```

Comment:
Would it be better to omit some of the error handling code from the explanation similar to the other language tab examples, or do you think it's better to include it? If the latter, maybe we could provide the users with more direction on how to resolve the error such as:
"Unable to connect to uri %v, check that your server is running on that host and port: %v"

Reply:
For the other languages, I tried to keep the code to a minimum by referring directly to the values that they should change in the CSFLE guide project. Could go the other way, but would need to change all of those examples.